identifiers/fields.txt

```diff
@@ -2,13 +2,20 @@ agilent.serial
 apache.info
 apache.variant
 apache.variant.version
+aptinex.model
+brocade.model
+brocade.submodel
 chromecast.capabilities
 chromecast.generation
 chromecast.serial_number
 cisco.imc_model
+cisco.model
 cisco.serial_number
 cookie
 dell.service_tag
+digi.serial
```

Comment:
If this stays (and IMO it should not, see comment below) then it needs to be `serial_number`, not `serial`.